modules/main/pom.xml

```diff
@@ -43,8 +43,8 @@
 
         <application.jersey.provider.paths>com.intuit.wasabi.api;com.codahale.metrics.jersey</application.jersey.provider.paths>
         <application.jersey.pojo.enabled>true</application.jersey.pojo.enabled>
-        <application.jersey.request.filters>com.sun.jersey.api.client.filter.LoggingFilter</application.jersey.request.filters>
```

Comment:
@iizrailevsky I tested it. Existing integration tests & unit tests work fine + no change in logging of messages. Actually, it was throwing a warning message earlier and ignoring it, when it was present - so, it was really not serving any purpose. :)